cumulus/ansible/tasks/playbooks/roles/ec2-pod/defaults/main.yml

```diff
@@ -1,26 +1,19 @@
 ---
+ec2_launch_instance_timeout: 1800 # 1/2 hour
 
-    cluster_name: pod
+terminate_wait_timeout: 600
 
-    region: us-east-1
+ssh_cidr_ip: "{{ source_ssh_cidr_ip | default('0.0.0.0/0') }}"
 
-    default_public_key: ec2
-    default_image: ami-d05e75b8
-    default_instance_type: t2.medium
+# Very insecure!
+ec2_pod_rules:
+  - proto: all
+    from_port: all
+    to_port: all
+    cidr_ip: "{{ ssh_cidr_ip }}"
 
```

Comment:
Actually the current logic is wrong,  I'm pushing a fix now - we should use the calculated defaults if ec2_pod_rules is not defined.